---
layout: default
title: Genes and DNA
parent: Fall
grand_parent: Biology
nav_order: 4
---

# Genes and DNA
{: .no_toc }

Fall Biology
{: .fs-6 .fw-300 }

---

<details open markdown="block">
  <summary>
    Table of contents
  </summary>
  {: .text-delta }
1. TOC
{:toc}
</details>

---

## 1.3: Life Processes Information and Requires Energy
### Introduction
- Sutton and Boveri proposed the **chromosome theory of inheritance.**
- Key point: **hereditary and/or genetic information is encoded in units called genes that are located on chromosomes** in cells. 
- Moleculular nature of genetic material later found.
- Chromosome consists of a molecule of DNA.

### The Central Dogma
- 1953 James Watson and Francis Crick proved DNA was a double-stranded helix.
- Each strand of a double helix is made up of four molecular building blocks.
- DNA carries, or encodes, information required for organism growth and reproduction.
- Base pairing allows DNA to be copied and preserves information encoded inside it.

<br>

- **The centrla dogma** describes the flow of information in cells.
- Dogma means a framework for understanding.
- DNA codes for RNA, which codes for proteins.

<br>

- Cells make a copy of a particular gene's infromation in the form of RNA, which carries out specialized functions in cells.
- Cell reads messenger RNA to determine what blocks to use to make a protein.
- Structure of DNA provided insight into genetic information passed when cells replicate.

<br>

- Mistakes in copying DNA leads to differences in sequences of building blocks in proteins.
- Outward appearance is a product of proteins, so DNA sequence difference may lead to differences in finch beak size and shape or length of a giraffe's neck.
- Heritable variations underlie the diversity of life.

### Life Requires Energy
- The chemical reactions that sustain life take place inside cells.
- Transmitting genetic information requires energy.
- Organisms are capable of living in many environments because they *vary in cell structure and in how they acquire and use energy.*

<br>

- Organisms need two things:
  - Chemical energy in the form of ATP
  - Moelcules that can be used as building blocks for synthesis of DNA, RNA, proteins, cell membrane, and other compounds.
- How do organisms obtain these materials?

<br>

- Plants and bacteria produce sugar using energy from sunlight.
- Use sugar to make ATP or store in other energy-rich molecules.
- Plants that are eaten or decomposed have their energy molecules obtained by animals, fungi, archaea, or othe rbacteria.
- **Crucial:** how do organisms secure food?
  - Vastly different methods of doing so.

<br>

- Cells in a multicellular organism are connected by common lineage.
- Third great founding idea in biology was that all distinct identifiable type sof organisms are connected by a common ancestry.

[Back to top](#)

---

## 15: DNA and the Gene: Synthesis and Repair
- DNA replication and repairing to preserve genetic information
  - 15.1 What are genes made of?
  - 15.2 DNA synthesis and early hypotheses
  - 15.3 Steps in replication - models for faithful DNA synthesis
- How can genes be copied and faithfully passed to offspring?

## 15.1: What Are Genes Made Of?
- Chromosome theory of inheritance proposed that chromosomes contain genes.
  - Biologists had known since the late 1800s that chromosomes in living cells were full of DNA and proteins.
- What are genes made of? - DNA or protein?

### Initial Biological Hypothesis That Genes Were Made of Proteins
- Initially, biologists believed genes were made of proteins.
- Limitless variation in structure and function, proteins seemed suited to contain information to program a cell.
- DNA was known to contain only 4 types of DNA building blocks.
- Early incorrect model of DNA suggests it wasa a monotonous and repetitive molecule.
  - Seemed impossible to hold complex information.

### The Herschey-Chase Experiment
- In 1952, Alfred hershey and Martha Chase took up the question.
- 8 years before the study, Oswald Avery et al. used bacterial cells to show DNA *could* serve as a genetic material.
- Herschey & Chase studied how a virus called T2 infects and replicates within bacterium *Escherichia coli* (inhabitant of human gut).
  - *E. coli* was tiny and grows quickly in the laboratory.
  - Was the favored model organism in studies of biochemistry.
- Herschey & Chase knew that T2 contained only DNA and proteins.
  - Somehow genes from the virus made their way to infected bacterial cells.
- After conducting experiments, Herschey & Chase came to a model of how T2 infects bacterial cells and replicates.
  - Exterior protein coat (*capsid*) of virus left behind on the exterior of the host cell.
- Strategy for determining composition of viral substance based on two facts:
  1. T2 proteins contain sulfur, but not phosphorus.
  2. DNA contains phosphorus, but not sulfur.
- Researchers grew viruses in presence of either radioactive isotop of sulfur or radioactive isotope of phosphorus.
  - Isotopes were incorporated into proteins and DNA.
  - Produced viruses with radioactive proteins and a population with radioactive DNA.
- Herschey & Chase allwoed radioactive viruses to infect *E. coli* cells.
  - If genes consisted of DNA, only genes would be injected in the cell, and radiactive protein would be only found on capsids and radioactive DNA would be found inside the cells.
- Herschey & Chase sheared capsids off cells by agitating cultures.
  - When samples were spun in a centrifuge, small virus capsids remained in the solution; larger cells formed a pellet at the bottom of the centrifuge.
- Biologists found that all radioactive protein was outside the cells; DNA was inside the host cells.
  - Newly created virus particles contained radioactive DNA *but not radioactive protein.*
  - The DNA component must represent the virus's genes.

#### Experiment in Step-by-Step Tabular Form

| Component | Description |
| --- | --- |
| Question | Do viral genes consist of DNA or protein? |
| DNA Hypothesis | T2 virus genes consist of DNA. |
| Protein Hypothesis | T2 virus genes consist of protein. |
| Experimental Setup | Label viruses (grow one set of TS w/ radioactive DNA and radioactive protein). Infect bacteria. Agitate cultures. Centrifuge solutions and force cells into pellet. Record location of radioactive labels. |
| Prediction of DNA Hypothesis | Radioactive DNA will be located within the pellet. |
| Prediction of Protein Hypothesis | Radioactive protein will be located within the pellet. |
| Results | Radioactive DNA is in pellet. Radioactive protein is in solution. |
| Conclusion | T2 virus genes consist of DNA. |

### The Structure of DNA
- Watson and Crick proposed a model for the structure of DNA one year after Herschey and Chase's results.
- DNA is double-stranded; each polymer strand is made of monomers.
- Deoxyribonucleotides link together to form a polymer through a phosphodiester linkage or bond.
- DNA has two features; a backbone and a series of bases that project from the backbone.
- Each molecule has *difference 3' and 5' ends*.
- Lined these two strands up in opposite directions in antiparallel fashion.

## 15.2: Testing Early Hypotheses About DNA Synthesis
- A-T and G-C pairing rules suggest a way for DNA to be copied when chomosomes are replicated.
- Proposed that existing strands of DNA served as a template for production.

### Three Alternative Hypotheses
- Biologists proposed some alternatives about DNA replication:

Hypothesis | Description
--- | ---
Semiconservative replication | If parental strands of DNA separate, each could be used as a synthesis of a new daughter strand. Each daughter DNA molecule consists of one old strand and one new strand. Conserves only one of the strands.
Conservative replication | If bases of strands turned out from the helix, they could serve as a template for an entirely new double helix all at once.
Dispersive replication | Parental double helix was fragmented into small pieces before replication, and each piece was replicated either with conservative or semiconservative mechanisms. Fragments would be joined into two molecules that contained a mixture of parental and daughter strands.

### The Meselson-Stahl Experiment
- Worked with *Escherichia coli* (*E. coli*) (same bacterium used in Hershey & Chase)
- Bacterial cells copy their entire complement of DNA (genome) before every cell division.
- Distinguish parent strands from daughter strands: plan to grow *E. coli* cells in the presence of nitrogen isotopes of different masses.
  - Different nitrogen isotope available in growth as DNA synthesized; the parental & daughter strands will have different identies.
  - Density-gradient centrifugation separates molecules based on their density. [^1]
  - Double stranded DNA molecules are subjected to density-gradient centrifugation; DNA that contains heavier isotopes of nitrogen will form a band lower in the tube.
- DNA containing nitrogen isotope-14 and nitrogne isotope-15 could be sseparated into two bands.
- Grew *E. coli* cells with nutrients containing only `15_N` (nitrogen isotope 15); purified DNA and transferred culture to growth medium with `14_N`.
- After enough time, cells divided (DNA copied). Culture divided again; sample was removed and DNA was isolated.
- Concluded that conservative replication and dispersive replication was wrong. 

#### Experiment in Step-by-Step Tabular Form

| Aspect | Description |
| --- | --- |
| Question | Is replication semiconservative, conservative, or dispersive? |
| Hypothesis 1 | Replication is conservative. |
| Hypothesis 2 | Replication is semiconservative. |
| Hypothesis 3 | Replication is dispersive. |
| Experimental Setup | Grow *E. coli* cells in medium with `15_N`. Transfer cells to medium with `14_N`. Let the cells divide twice (total 3 generations). Centrifuge the three samples separately. Compare locations of DNA bands.
| Prediction 1 | After two generations: 1/2 low-density DNA, 1/2 intermediate-density DNA. |
| Prediction 2 | 1/4 high-density DNA, 3/4 low-density DNA. |
| Prediction 3 | All intermediate-density DNA. |
| Results | After 2 generations, 1/2 low-density DNA and 1/2 intermediate-density DNA. |
| Conclusion | Replication is semiconservative. |

## 15.3: A Model for DNA Synthesis
- DNA inside a cell is like an ancient cell.
- Ancient tests contain messages thousands of years old; DNA in cells has been copied and passed down from LUCA.

### DNA Polymerase
- DNA replication breakthrough: discovery of DNA polymerase.
  - Polymerizes deoxyribonucleotide monomers into DNA.
  - Is a protein that catalyzes DNA synthesis.
  - Many different types of DNA polymerase.
- DNA polymerase only works in one direction; only adds deoxyribonucleotides to the 3' end of the chain.
  - DNA synthesis goes from 5' to 3'.

#### Endergonic Polymerization Reactions
- Polymerization reactions are usually endergonic (require input of energy).
- DNA synthesis requires an input of energy, but potential energy of deoxyribonucleotide monomers is raiased by reactions tha add two phosphate groups, forming **deoxyribonucleotide triphosphates (dNTPs)**.
  - *n* represents any of 4 bases found in DNA.
  - dNTP has high potential energy; forms phosphodiester bonds in DNA strand.

### Where Does Replication Start?
- Biologists used electron microscopy to catch DNA replication in action.
- Replication bubble forms as DNA is synthesized.
  - Forms at sequence of bases called *origin of replication.*
- Bacterial chromosomes have only one origin of replication. Eukaryotes have multiple origins of replication.
- **Replication fork** is a Y-shaped region where parental DNA double helix is separated into strands and copied.
- DNA synthesis occurs in two directions at once.

### How is the Helix Opened and Stabilized?
- Proteins converge at an origin of replication in bacteria and eukaryotes.
- However, organisms use different ways to control DNA synthesis.

| Type | Process |
| --- | --- |
| Bacteria | Set of proteins recognizes origin of chromosome; strands near origin are separated. DNA helicase (enzyme) binds to one of the single strands of DNA near the forks. |
| Eukaryotes | Proteins that initiate DNA replication are held tight by cell-cyle regulatory proteins. DNA helicase is loaded onto double-stranded DNA at the origin. |
| All organisms | DNA helicase uses energy of ATP hydrolysis (breakdown) to separate strands of DNA at replication forks. **Single-stranded DNA-binding proteins (SSBPs)** attach to strands to prevent them from becoming double helices again. |

- Topoisomerase is an enzyme that cuts DNA, allows it to unwind, and rejoins it.
- Acts as DNA replication works ahead of the advancing replication fork to relieve twists generatd by DNA helicase.

### How is the Leading Strand Synthesized?
1. **DNA is opened, unwound, and primed.**
  - Primase synthesizes the RNA primer. 
  - Topoisomerase relieves twisting forces.
  - Helicase opens double helix.
  - SSBPs stabilize large strands.
2. Synthesis of leading strand begins. 
  - Sliding clamp holds DNA polymerase in place.
  - DNA polymerase synthesizes leading strand in 5' to 3' direction.

- Consequences of anti-parallel nature of DNA strands and limitations of DNA polymerases.
  1. DNA polymerase can synthesize DNA only in 5' to 3' direction.
  2. DNA polymerase cannot start synthesizing from scratch on a template strand.
    - Can only extend fromt he 3' of an existing strand that is hydrogen-bonded to the template.
- 3' end supplied by a strand of RNA called a **primer** that is base paired to the DNA template.
  - Primase is a RNA polymerase (enzymes that catalyze polymerization of ribonucleotides into RNA).
  - RNA polymerase can start synthesis from scratch.
- Primer laid down on single-stranded template.
  - DNA polymerase adds deoxyribonucleotides in the 5' to 3' direction.
  - A protein ring encircles DNA and binds to DNA polymerase.
  - Sliding clamp keeps DNA polymerase from falling off the DNA during synthesis.
- **Antiparallel orientation of DNA** is significant.
  - Strand of DNA synthesizes towards replication fork called **leading strand** (or **continuous strand**), because its synthesis proceeds continuously  in the direction of the moving replication fork.
- After RNA primer is in place, DNA polymerase moves along reading the template and adding deoxyribonucleotides one by one to the extending 3' end of the strand.

### How is the Lagging Strand Synthesized?
- Lagging strand (discontinuous strand) is synthesized in the direction away from the replication fork.

#### Discontinuous Replication Hypothesis
- Proposed to explain how lagging strand was synthesized.
- Primase synthesizes new RNA primers for lagging strands as the moving replication fork opens single-stranded regions of DNA.
  - DNA polymerase syntheses short DNA fragments from these primers.
  - Fragments linked to form a continuous strand.

#### Reiji and Tsuneko Okazaki Experiment
- 1960s; tested a prediction of discontinuous replication hypothesis - existence of short DNA fragments on one of two newsly synthesized strands of DNA.
- Pulse-chase strategy.
- Exposed a culture of growing *E. coli* cells to radioactive deoxyribonucleotides (pulse) and transferred cells to growth medium with nonradioactive deoxyribonucleotides (chase).
  - Radioactive deoxyribonucleotides should first appear in short fragments of DNA.

#### The Discovery of Okazaki Fragments
- Researchers found short radioactively labelled DNA fragments averaging about 1000 deoxyribonucleotides long.
- Short DNAs (attached to RNA primers) known as **Okazaki fragments**.
- Gradually became longer during the chase.

#### Connection of Okazaki Fragments
- Bacteria: DNA polymerase III dissasociates from 3' end of Okazaki fragment when polymerase encounters RNA primer that begins next Okazaki fragment.
- DNA polymerase I attaches to 3' end of Okazaki fragment.
  - Moves in 5' to 3', removing the RNA primer ahead of it and replaces ribonucleotides with deoxyribonucleotides.
- Enzyme called DNA ligase catalyzes the formation of the phosphodiester bond between 3' and 5' ends of adjacent Okazaki fragments, closing the backbone.

### New Discoveries in DNA Synthesis
- New insights continue to emerge.
- Proteins and enzymes work in a macromolecular machine called the **replisome.**
- Replisome may contain up to three copies of DNA polymerase III.

## 16.1: What Do Genes Do?
- George Beadle and Edward Tatum published a series of important experiments.
  - Idea: knock out a gene by damaging it and then infer what the gene does by observing the phenotype of the mutant individual.
- Alleles that do not function are *null alleles*.
- This strategy is one of the most effective research strategies.

### The One-Gene, One-Enzyme Hypothesis
- Beadle & Tatum used the bread mold *Neurospora crassa.*
- Exposed *N. crassa* to radiation, which can damage DNA.
  - Inability to produce pyridoxine (vitamin B6) was due to a defect in one gene.
  - Failure to synthesize other genes were due to defects in those genes.
- **One-gene, one-enzyme hypothesis:** mutants could not make a compound because it lacked an enzyme to synthesize it.
  - Proposed that the lack of enzyme was due to genetic defect.
  - Proposed that each gene contains information needed to make an enzyme.
  
### An Experimental Test of the Hypothesis
- Adrian Srb and Normal Horowitz published a test of the one-gene, one-enzyme hypothesis.
- Biologists focused on the ability of *N. crassa* to synthesize amino acid arginine.
  - Normal cells grow well on a medium without arginine.
  - Wild-type cells produce their own arginine.
- Arginine is synthesized in a **metabolic pathway**.
  - Ornithine and citrulline are intermediate products on the metabolic pathway leading to arginine.
- Srb & Horowitz hypothesized that particular *N. crassa* genes are responsible for producing the three enzymes in the metabolic pathway.
- Srb & Horowitz used radiation to create mutant cells.
- **Genetic screen** - a technique for picking out specific types of mutants.

#### Tabular Form

| Aspect | Description |
| --- | --- |
| Question | What do genes do? |
| Hypothesis | Each gene contains information needed to make one enzyme. |
| Null Hypothesis | Genes do not have a one-to-one correspondence with enzymes. |
| Experimental Strategy | Produce mutants unable to synthesize arginine, then test different steps in the mtabolic pathway for synthesizing arginine. |
| Experimental Setup | Isolate *Neurospora crassa* that cannot synthesize arginine in four environments: no supplement, supplemented with only ornithine, supplemented only with citrulline, supplemented only with arginine |
| Prediction of Hypothesis | There will be 3 distinct types of mutants corresponding to defects in enzyme 1, 2, and 3 in the pathway for synthesizing arginine. Each mutant will be able to grow  on different combinations. |
| Prediction of Null Hypothesis | There will not be a simple correspondence between a particular mutation and a particular enzyme. |
| Results | Three types of mutants *arg1*, *arg2*, *arg3*. They are able to grow in the metabolic pathway *from* orthinine, citrulline, and arginine, respectively. *arg1* cells lack enzyme 1 from precursor to orthinine, *arg2* cells lack enzyme 2 from orthinine to citrulline, etc. |
| Conclusion | The one-gene, one-enzyme hypothesis is supported. |

## 16.2: The Central Dogma of Molecular Biology
- How does DNA code the production of enzymes?
- Part of the answer: structure of DNA.
- Watson and Crick's model offered little hope that DNA could directly catalyze reactions.
  - DNA does not allow it to bind to many substrates needed for protein synthesis.

### The Genetic Code Hypothesis
- Francis Crick proposed that the sequence of bases in DNA acted like a code.
  - DNA was only for information storage.
  - It would need to be read and used to produce proteins.
- Different combinations of bases could specify the 20 amino acids (like different dots and ashes in Morse code specify the 26 letters of the alphabet).
- Stretch of DNa can contain information needed to build an amino acid.

### RNAas the Intermediary between Genes and Proteins
- DNA must go through an intermediary structure to produce proteins.
  - Understood from cell structure: DNA is enclosed in the nucleus, but ribosomes (where protein synthesis happens) lies outside of the nucleus.
- Francois Jacob and Jacques Monod suggested RNA molecules linked genes to the protein-manufacturing centers in the cytoplasm.
- Single-stranded RNA (messenger RNA or mRNA) carry information out of the nucleus from DNA to protein synthesis sites in the cytoplasm. 
  - **RNA polymerase can synthesize this complementary RNA molecule.**
  - Does not need a primer to begin connecting ribonucleotides together.

### Dissecting the Central Dogma
- Central dogma: DNA (information storage) to RNA (information carrier) to Proteins (cell machinery)
  - The flow of information between molecules (not conversion of one molecule into another).

#### The Roles of Transcription and Translation
- **Transcription** is using a DNA template to make an RNA molecule. Transcribed with RNA polymerase.
- **Translation** is the process of using information in the base sequence of mRNA to synthesize porteins. Information in mRNA is translated into proteins by ribosomes.

#### Linking Genotypes to Phenotypes
- An organism's genotype is determined by the sequence of bases in its DNA.
- Its phenotype is the product of the proteins it produces.
- Extension of Crick's idea: DNA (genotype) to mRNA to proteins to phenotype.
- Each DNA sequence contributes to an **allele** (form) of the gene.
  - The result of these two different DNA sequences is the production of proteins that differ in amino acid sequence.

#### Modifications of the Central Dogma
- Highlights of new discoveries:
  - Many genes code for RNA molecules that do not function as mRNAs (transcribed from DNA but never translated into proteins), but can control many important phenotypes.
  - Information sometimes flows from RNA to DNA.
- Discovery of 'reverse' information: group of viruses whose genes consisted of RNA.
  - Reverse transcriptase synthesizes a DNA version of the genes.
- Central dogma is important, but life is not always as clean.

## 16.3: The Genetic Code
- Once biologists understood the pattern of information flow, they asked: how can the base sequence of mRNA code become amino acids?
- Genetic code: the relationship between a sequence of nucleotides in DNA or RNA and the seuqence of amino acids in a protein.

### How Long is a Word in the Genetic Code?
- Genetic language, words specify amino acids. How long is the message that designates one amino acid?
- George Gamow: each genetic word contains 3 bases.
  - There are only 4 different bases (A, U, G, C). A one-base code can specify only 4 different amino acids.
  - A two-base code can represent 16 different amino acids.
  - A three-base code can represent 64 different amino acids.
- Three-base code (**triplet code**) is shortest genetic code for at least 20 amino acids.
  - The genetic code may be redundant (amino acid can be specified by more than one triplet of bases).
- **Codon** - group of three bases that specifies an amino acid.
  - Different codons might mean the same amino acid (e.g. AAA and AAG).

#### Francis Crick and Sydney Brenner Experiment
- Developed an experiment to see if codons were actually 3 bases long.
- Used chemicals that caused an occasional addition or deletion of a base pair.
- This led to the loss of function in the gene being studied.
- A single addition or deletion throws the sequence of codons (**reading frame**) out of register.
- Functional proteins were produced when sequences were eliminated when three base pairs were eliminated altogether.

### How Did Researchers Crack the Code?
- Marshall Nirenberg and Heinrich Matthaei developed a method of synthesizing RNAs composed of single-type of ribonucleotide.
  - RNA triplet UUU codes for phenylalanine.
  - RNA triplet AAA codes for lysine.
  - RNA triplet CCC codes for proline.
- Extended this work to create RNAs from different mixtures of different nucleotides.
  - Researchers could predict how often a paticular triplet woul occur on average in RNA molecules.
  - Ribosome-binding experiments enabled Nirenberg & Leder to fill in the meanings of all remaining unknown codons.
- **Some codons do not specify amino acids.**
  - Instead, they signal an end fo the reading frame.
- **Start codons**. AUG start codon signals protein synthesis should began at that point.
- **Three stop codons** (termination codons) with sequences UAA, UAG, and UGA. Stop codons signal for the end fo a reading frame (end of a polypeptide).

#### Complete Genetic Code Table
![Complete Genetic Code](https://andre-ye.github.io/biology/genes-and-dna/codon%20base%20table.PNG)

#### Analyzing the Code
- Important properties:

| Property | Description |
| --- | --- |
| Redundant | All amino acids (except for 2) are coded for by more than one codon. Codons specifying the same amino acid are synonymous codons. |
| Unambiguous | A given codon never codes for more than one amino acid. | 
| Non-overlapping | Once the ribosome locks onto the first codon, the reading frame is established and the ribosome rads one codon one after another. |
| Universal | With a few exceptions, codons specify the same amino acids in all organisms. |
| Conservative | Several codons specify the same amino acid, but the first two bases are usually identical. |

- If a change in DNA sequence leads to a change in the third position, it is less likely to alter the amino acid in the protein.
- Genetic code minimizes the phenotypic affects of small alterations.
  - Genetic code not assembled randomly; honed by natural selection and is remarkably efficient.

#### The Value of Knowing the Code
- Knowing the genetic code lets biologists predict the amino acid sequence encoded by a particular DNA sequence.
  - Lets them determine mRNA and DNA sequences that could code for a particular set of amino acids.
- Set of mRNA or DNA sequences can code for a particular amino acid sequence.
  - Code is redundant.
  - If a polypeptide contains phenylalanine, don't know if it is UUU or UUC.
  
## 16.4: What Are the Types and Consequences of Mutation?
- Mutations change genetic information.
- A mutation was defined as a heritable change in a gene.
- Heritability can be from mother cell ot daughter cells or between generations of multicellular organisms.
- **Molecular view of mutation:** mutation is any permanent change in the organism's DNA.
- Is a modification of a cell's information archive (changes its genotype and creates new alleles).

### Point Mutations
- A change in the sequence of bases can result to an alteration of one or a small number of base pairs: a *point mutation.*
- Point mutations that change the identity of an amino acid in a protein are **missense mutations.**
- A point mutation that has no impact on the amino acid is a **silent mutation.**
- Some mutations can shift the reading frame, and are called **frameshift mutations.** Usually destroy the function.
- **Nonsense mutation** occurs when a codon that specifies an amino acid is changed into one that specifies a stop codon. This causes early termination of the polypeptide chain.

![Types of point mutations](https://andre-ye.github.io/biology/genes-and-dna/types%20of%20point%20mutations%20table.PNG)

#### The 3 Categories of Mutations
1. **Beneficial.** Some mutations increase the fitness of an organism.
2. **Neutral.** If the mutation has not effect on fitness, it is neutral. e.g. silent mutations.
3. **Deleterious.** Most individuals are well-adapted to their current habitat and mutations are random changes in genotype; most mutations lower fitness.
- **Most point mutations are slightly deleterious or neutral.**

### Chromosome Mutations
- There are larger-scale mutations that can change the entire structure or number of chromosomes.
- A polyploidy is the state of having more than two of each type of chromosome, an aneuploidy results from the addition or deletion of individual chromosomes.
- Changes in chromosomes result from mistakes in moving chromosomes during meiosis or mitosis.
  - Mutations don't change DNA sequences but cause permanent changes in an organism's DNA.
- Structure of individual chromosomes can be changed in significant ways.
- Four major types:
  1. A **deletion** can be caused by a broken segment of a chromosome.
  2. An **inversion** can be caused when segments of a broken chromosome are flipped and re-joined.
  3. Errors in crossing over or in DNA synthesis lead to the presence of a **duplication**, or multiple copies of the segment.
  4. **Translocation** occurs when a broken piece of a chromosome is attached to a different chromosomes.
- Chromosome mutations can be beneficial, neutral, or deleterious.

## 17.1: An Overview of Transcription
- The first step in using genetic information is transcription.
  - This is the synthesis of an RNA version of the instructions in archive.
- Enzymes called **RNA polymerases** are important in transcription.
  - RNA polymerases use monomers called ribonucleotide triphosphates (dNTPs) used for DNA synthesis.
- These are like dNTPs used for DNA synthesis but have a hydroxyl group at the 2' carbon.
- Once an NTP is in place, the RNA polymerase catalyzes a reaction that cleaves off the two phosphates and forms a phosphodiester linkage.
  - Forms between the 3' end of the growing mRNA chain and the new ribonucleoside monophosphate.
- RNA that is complementary to one of the DNA strands is synthesized in 5' to 3' direction.
- Terminology:
  - The strand read by RNA polymerase is the **template strand.**
  - The other strand is the **coding strand.**
    - The RNA sequence matches it, because it is synthesized in the same direction.
- RNA polymerase does not require a primer to begin transcription.
- Bacteria have a single RNA polymerase; eukaryotes have at least three distinct types.

### Initiation: How Does Transcription Begin in Bacteria?
- One way of thinking about a gene: a stretch of DNA that is transcribed to produce a functional product for a cell.
  - How does RNA polymerase find genes?
- **Initiation** is key to control.
- **Core enzyme**, which transcribes the genes, needs **sigma** protein to bind to the core enzyme to recognize sites where transcription should begin.
  - Sites are called **promoters**.
- Bacterial RNA polymerase core enzyme and sigma form a **holoenzyme** ('whole enzyme').

#### Bacterial Promoters
- David Pribnow: offered an initial answer in the mid-1970s.
- Pribnow found that promoters were 40-50 base pairs long and had a series of bases identical or similar to TATAAT.
  - Known as the -10 box, because it is centered about 10 bases from the point where transcriptionb egins.
- RNA polymerase moves *downstream* from a point of reference, DNA is said to be *upstream* from the point of reference.
- The sequence TTGACA is 35 bases upstream of the +1 site called the -35 box.

#### Events Inside the Holoenzyme
- Transcription can only be initiated when sigma binds to the -35 and -10 boxes in DNA.
- The sigma protein can bind the promoter in only one orientation.
- The orientation of the promoter determines which DNA strand will be used as the template and which direction RNA polymerase will start synthesizing.
- When the holoenzyme is bound to a promoter,
  1. RNA polymerase opens the DNA helix.
  2. DNA strands form a 'transcription bubble'.
- Ribonucleoside triphosphates (NTPs) enter a channel in the enzyme and diffuse to the active site.

### Elongation and Termination in Bacteria
- When the RNA polymerase leaves the promoter region as it synthesizes RNA, the **elongation** phase is under way.
- During elongation, the enzyme reads the DNA template on the 3' end at a rate of 50 nucleotides per second.
- RNA polymerase proofreads and corrects errors.
  - A macromolecular machine with a structure critical for its function.
- Differerent parts of the enzyme helped steer the template through channels inside the enzyme and to separate newly synthesized RNA from the DNA template.
- Double-stranded DNA goes in and out of one grove.
  - NTPs enter another one.
  - RNA strand exits out the rear.
- **Termination** ends transcription.
  - Transcription stops in bacteria when RNA polymerase transcribes a DNA sequence called a transcription-termination signal.

### Transcription in Eukaryotes
- Eukaryotes are somewhat similar in features of transcription to bacteria.
- Important differences:
  - Eukaryotes have **three** major polymerases.
    - RNA polymerase I, II, & III (pol I, pol II, pol III).
    - Each polymerase produces *certain types of RNA*. 
  - Promoters in eukaryotic DNA are larger and  more diverse.
    - Most eukaryotic promoters include a 'TATA box', centered 30 base pairs upstream of the transcription start site.
  - Eukaryotic RNA polymerases recognize promoters using **general transcription factors.**
    - First to assemble at the promoter, RNA polymerase follows.
  - Termination of transcription differs from termination in bacteria.
    - DNA sequence near the end of each gene called the polyadenylation signal (**poly(A) signal**) is transcribed.
    - RNA downstream continues to transcribe the DNA template.
    - RNA polymerase comes off the DNA template.
  - Transcription and translation are separated in time and space. 
    - Transcription occurs in the nucleus and translation occurs in ribosomes of the cytoplasm for eukaryotes.
    - Bacteria begins translating mRNA before its transcription is complete.
    
### Process: Initiating Transcription in Bacteria
1. Initiation begins. Sigma binds to the promoter region of the DNA, characterized by the -35 box and the -10 box.
2. Initiation continues. RNA polymerase opens up the DNA helix and transcription begins. NTPs are used to create the complement of the template strand. 
3. Initiation completes. Sigma is released from the core enzyme and RNA synthesis continues. The RNA polymerase moves downstream along the DNA.

### Process: Ending Transcription in Bacteria
1. Hairpin forms. RNA polymerase transcribes a transcription-termination signal, which codes for RNA that forms a hairpin. 
2. Termination. The RNA hairpin leads to the RNA separating from RNA polymerase, terminating transcription.

## 17.2: RNA Processing in Eukaryotes
- Newly transcribed eukaryotic RNAs were nonfunctional and many times larger than corresponding RNAs.
- Eukaryotic genes are initially copied from nonfunctional RNAs called **primary transcripts**.
  - Primary transcript is pre-mRNA for protein-coding genes.
- Primary transcript required **RNA processing** to generate mature and functional RNA.

### The Startling Discovery of Split Eukaryotic Genes
- Richard Roberts and Phillip Sharp (1977) discovered a common-cold virus had protein-coding genes with intervening sequences of noncodingDNA.
  - Key information *was split into pieces.*
- Experiment:
  1. Heat the virus's DNA to break hydrogen bonds.
  2. Single-stranded DNA incubated with mRNA.
    - Intention: promote base pairing to reveal where genes occurred.
- Parts of the DNA formed loops.
  - Stretches of DNA not represented in corresponding mRNA.
- Example: eukaryotic genes do not carry messages like
```
Biology is my favorite course of all time.
```
- Instead, the message would be something like
```
BIOLηεπpoενχνσoφγενεσOGY IS MY
FAVORαpεLντεppeπITε3vνoν中o命ωγ命ITECOURSE
OFαν{r可αωεTOl3εσπλLXTOγεTηεp ALL TIME
```
- Sections of a noncoding sequence are represented with random characters.
  - Must be removed from the RNA before it can carry an understandable message.

#### Introns and Exons
- Regions of a gene that are transcribed but not represented in the final RNA are introns.
- Regions that are transcribed and represented in final RNA are called exons (*ex*pressed in mature DNA).
- "intron" and "exon" also used for regions of transcribed RNA that are cut out (introns) or retained (exons).
- Exons are not necessarily protein-coding regions.
  - Many are, but some exons are RNA sequences that do not code for proteins.

### RNA Splicing
- Transcription of eukaryotic genes generates a primary transcript with exons and introns.
- Through **splicing**, introns are removed from the growing RNA strand.
  - Pieces of the primary transcript are removed and remaining segments are joined.
  - Results in uninterrupted RNA mesage.
- Protein + RNA macromolecular machiens are small nuclear ribonucleoproteins (snRNPs).

#### The Process of RNA Splicing
1. snRNPs (small nuclear ribonucleoproteins) bind to the start and end of an intron and to a branch site within the intron.
2. More snRNPs join, and a spliceosome assembles. These are large macromolecular machines that form as an aggregate of snRNPs.
3. The 5' end of the intron is cut from the exon, and the intron forms a single-stranded stem with a loop (a lariat) with an adenine in its connecting branch point.
4. The 3' end of the intron is cut, releasing the intron as a lariat. Exons are joined by a phosphodiester linkage. Cut intron is degraded to ribonucleoside monophosphates.

### Adding Caps and Tails to Transcripts
- Two additional processing events are required for splicing.
  1. When 5' end of pre-mRNA emerges, enzymes add a 5' cap.
    - Modified guanine nucleotide linked to transcript in an unusual way.
    - Enables ribosomes to bind to the mRNA and protects the 5' end of the mRNA from enzymes that degrade RNA (ribonucleases).
  2. Enzyme cuts the 3' end of the pre-MRNA after the poly(A) signal.
    - Specialized RNA polymerase adds 100 to 250 adenine nucleotides.
    - Known as the poly(A) tail.
    - Not encoded by the DNA template strand.
    - Required for ribosomes to start translation and to protect the end of mRNA from attack by enzymes.

![](https://andre-ye.github.io/biology/genes-and-dna/pre%20mrna%20diagram.PNG)

## 17.3: An Introduction to Translation
- To synthesize a protein, the sequence of bases in a messenger RNA molecule must be translated into a sequence of amino acids in a polypeptide.

### Ribosomes Are the Site of Protein Synthesis
- *Where does translation occur?*
- Answer: observation that there is correlation between number of ribosomes in a cell and rate of protein synthesis.
  - Test: Roy Britten et al. performed a pulse-chase experiment.
  - Concluded that proteins are synthesized at ribosomes and then released.

### An Overview of Translation
- Decade after ribosome hypothesis confirmed: electron micrographs showed bacterial ribosomes in action.
  - Bacteria: ribosomes can attach to mRNA and begin synthesizing proteins before transcription was complete.
  - Multiple ribosomes attach to each mRNA.
    - Structure of this is a polyribosome.
  - Polyribosomes increase the number of copies of a protein that can be made from single mRNA.
- Transcription and translation can be coupled in bacteria because there is no nuclear envelope to separate the two processes.
  - Gene expression can hence be very fast.
- Eukaryotes, transcription and translation are separated by *time and space.*
  - When mRNAs are outside of the nucleus, ribosomes can attach and begin translation.

  
### How Does mRNA Specify Amino Acids?
- Discovery of genetic code revealed that triplet codons in mRNA specify particular amino acids.
  - How does this conversion happen?
  
#### Two Hypotheses of Codon Interactions with Amino Acids
- **Hypothesis 1:** Bases in a particular codon were complementary in shape or charge to the side group of an amino acid.
  - Problem: how could nucleic acid bases itneract with nonpolar amino acid side groups?
- **Hypothesis 2:** Adapter molecules hold amino acids in place while interacting directly.

## 17.4: The Structure and Function of Transfer RNA
- Crick's ideas for an adapter molecule were discovered by accident.
- Ribosomes provide the catalytic machinery, mRNAs contribute the message to be translated, amino acids are the building blocks of proteins, and ATP supplies energy.
  - **Not enough.** Transfer RNA (tRNA) was needed to complete the picture.
- tRNA has an amino acid attached, known as **aminoacyl tRNA**.

### Zamecnik Experiment
- What happens to amino acids bound to tRNAs?
- Paul Zamecnik et al tracked radioactive leucine molecules attached to tRNAs; found that amino acids were transferred from tRNAs to proteins.

| Aspect | Description |
| --- | --- |
| Question | What happens to amino acids attached to tRNAs? |
| Hypothesis | Aminoacyl tRNAs transfer amino acids to growing polypeptides. |
| Null Hypothesis | Aminoacyl tRNAs do not transfer amino acids to growing polypeptides. |
| Experimental Setup | Attach radioactive leucine molecules to tRNAs. Attach aminoacyl tRNAs to an in vitro system that allows protein synthesis. Follow the path of radioactive amino acids. |
| Prediction of Hypothesis | Radioactive amino acids will be found in the polypeptides. |
| Prediction of Null Hypothesis | Radioactive amino acids will not be found in the polypeptides. |
| Results | Radioactive signal of tRNA and polypeptides are inversely proportional to each other. When there is low radioactive signal in tRNA, there is high radioactive signal in the polypeptides, and vice versa. |
| Conclusion | Aminoacyl tRNAs transfer amino acids to growing polypeptides. |

### What is the Structure of tRNAs?
- tRNA serves as a chemical go-between that allows amino acids to interact with an mRNA template.
- Initial studies established the sequence of nucleotides in tRNAs.
  - Usually short (75-95 nucleotides).
  - Certain parts of each tRNA could form hydrogen bonds with complementary base sequences somewhere else in the same molecule.
    - Forms stem-and-loop structures.
- **Two important parts of tRNA:**
  - CCA sequence at 3' end of tRNA molecule is the site for amino acid attachment.
  - Loop opposite to the amino acid attachment site contains three ribonucleotides that serve as an **anticodon.**
    - Anticodon is a triplet of ribonucleotides that can form base pairs with the codon for the amino acid in mRNA. e.g. anticodon os AAA is UUU.
- X-ray crystallography revealed tertiary structure of tRNA; folds into an L-shaped molecule with an anticodon on one end and the CCA sequence + attached amino acid on another end.

### How Are Amino Acids Attached to tRNAs?
- Three important components of linking amino acids to tRNAs:
  - Input of energy from ATP required to attach amino acid to tRNA.
  - Enzymes called **aminoacyl-tRNA synthetates** catalyze the addition of amino acids to tRNAs. Known as 'charging' a tRNA.
  - For each of the 20 major amino acids, there is a different aminoacyl-tRNA synthetase and one or more tRNAs.
- Each synthetase has a binding site for one amino acid and tRNA.
  - Differences in tRNA shape let enzymes recognize and match tRNA to the amino acid.
- The tRNA and aminoa cid structures fit together.
  - Precision is important; frequent mistakes in choosing the matching amino acid are disasterous.

### How Many Types of tRNAs Are There?
- Paradox: genetic code specifies 20 common amino acids using 61 different codons.
  - There should be 61 different tRNAs to read codons.
  - Most cells have about 40.
- How can 61 codons be translated with 40 tRNAs?
- **Wobble hypothesis:**
  - (Most) Amino acids are specified by more than one codon.
  - Codons for the same amino acid often have the same nucleotides for the first and second positions but different nucleotides in the third position.
  - `CAA` and `CAG` both code for glutamine; tRNA with GUU anticodon can pair with the first two bases of both.
  - tRNA reads the codon because the `U` in anticodon third position can form a nonstandard base pair with `G` in `CAG`.
  - Crick: proposed that certain bases bind to bases that do not match Watson-Crick base pairing.
    - Flexibility 'wobble' in base pairing.
  - Wobble pairing allows one tRNA to read more than one codon.
- Wobbles in the third position of a codon explain 40 tRNAs translating 61 arrangements of codons.
- **Wobble pairing does not account for redundancy**, it only explains how on tRNA can read more than one codon, not how one amino acid can be specified by more than one codon.

## 17.5: Ribosome Structure and Function in Translation
- Translation of each codon in mRNA into the next amino acid *begins when the anticodon of tRNA binds to the codon.*
- Codon translation is complete when a peptide bond forms between the amino acid and the polypeptide.
- rRNAs (ribosomal RNAQs) that, with proteins, make up ribosomes.
- Ribosomes separate into a large and small subunit.
  - Each ribosome consists of many rRNA molecules and proteins.
  - **Small subunit hold mRNA, large subunit is where peptide bonds are fromed.**
- Three sites:
  - tRNA carries an amino acid. This site is the 'A' site (acceptor or aminoacyl).
  - tRNA in the middle holds the growing polypeptide chain and occupies the P (peptidyl) site in the ribosome. Think as 'P' for peptide-bond formation.
  - tRNA that no longer has amino acids attached leave the ribosome, occupying the E site ('E' for exit).
- Process of protein synthesis in ribosomes:
  1. Aminoacyl tRNA diffuses into the A site. If its anticodon matches a codon in mRNA, it stays in the ribosome.
  2. A peptide bond forms between the amino acid held by the aminoacyl tRNA in the A site and the growing polypeptide in the P site.
  3. The ribosome moves relative to the mRNA by one codon. All three tRNAs are shifted one position within the ribosome.
    - tRNA in E site exits.
    - tRNA in P site moves to the E site.
    - tRNA in A site switches to P site.
    - A site is empty and ready to accept another aminoacyl tRNA.
- Protein being synthesized grows by one amino acid each time.
  - Protein synthesis begins at the amino end (N-terminus) of the polypeptide and proceeds towards the carboxy end (C-terminus).
  
### Initiating Translation
- To translate an mRNA, a ribosome must begin at the first codon in a message, translate the mRNA up to the termination, then stop.
- Three phrases of protein synthesis: initiation, elongation, and termination.
- Start codon is found near the 5' end of mRNA; codes for amino acid methionine.
  - Distinguish initiation of translation at the start codon from initiation of transcription at the promoter.
- Translation *begins when a **section of rRNA in a small ribosomal subnit binds to a complementary sequence on an mRNA.***
  - This mRNA region is the **ribosome binding site** (also called the **Shine-Dalgarno sequence**).
  - About six nucleotides upstream from the start codon.
- Interactions betweenthe small subunit, message, and tRNA are mediated by **initiation factors**.
  - Help prepare the ribosome for translation and in binding the first aminoacyl tRNA.
  - Modified form of methionine - N-formylmethionine (abbreviated f-Met).
- In **eukaryotes**:
  - More initiation factors are needed.
  - Ribosome first associates with cap on the 5' end of mRNA.
  - Initiator tRNA carries normal methionine.
- Initiation is complete when the large subunit joins the complex.
  - Initiator tRNA occupies the P site. This is the only time a tRNA carrying a single amino acid occupies the P site.
- Summary:
  - Translation initiation in bacteria is three-pronged.
    1. mRNA binds to a small ribosomal subunit.
    2. Initiator tRNA with f-Met binds to the start codon.
    3. Large ribosomal subunit binds.
  - Initiation in eukaryotes is different but has similarities.

#### Process of Initiating Translation in Bacteria
1. **mRNA binds to a small subunit.** A sequence in mRNA called the ribosome binding site binds to a complementary seuqnece in an RNA molecule, which is part of the small subunit of the ribosome. This is helped by initiation factors.
2. **Initiator aminoacyl tRNA binds to the start codon.** This usually carries f-Met.
3. The **large subuit of the ribosome binds**, completing the ribosome assembly; translation can begin.

### Elongation: Extending the Polypeptide
- At the start of elongation, the E and A sites in the ribosome are empty of tRNAs.
- An mRNA codon is exposed in the A site.
- The elongation phase begins when an aminoacyl tRNA binds to the codon in the A site.
- See process for more details on the elongation phase.

#### Is the Ribosome and Enzyme or a Ribozyme?
- Ribosomes contain roughly equal amounts of protein and RNA.
- The active site consisted entirely of RNA; therefore the ribosome is a ribozyme (not a protein-based enzyme).

#### Moving Down the mRNA
- Peptide bond formation involves the transfer of the amino acid linked to the tRNA in the P site to the amino acid held by the tRNA in the A site.
- During **translocation**, the ribosome moves one codon down from the mRNA once a new peptide bond is formed.
  - An mRNA in reality is ratcheted througha  stationary ribosome.
  - Important point: translocation is a codon-by-codon movement.
_ **Elongation factors** are proteins required for translocation.
  - Translocation requires energy; elongation factors bind to the ribosome and break down GTP, an energy-rich molecule.
- Anticodons of tRNA are bound to the codons of mRNA; movement of ribosome brings uncharged tRNA into the E site and the tRNA containing the growing polypeptide in the P site.

#### Process of Elongation Phase of Translation
1. **Incoming aminoacyl tRNA** moves into the A site. Its anticodon base pairs with the mRNA codon.
2. **Peptide bond formation**. The amino acid attached to the tRNA in the P site is transferred by formation of a peptide bond to the amino acid of the tRNA in the A site.
3. **Translocation**. The ribosome moves one codon down the mRNA with the help of elongation factors. THe tRNA attached to the polypeptide moves into the P site. The A site is empty.
4. **Incoming aminoacyl tRNA**. A new charged tRNA moves into the A site, where its anticodon base-pairs with an mRNA codon.
5. **Peptide-bond formation.** The polypeptide chain attached to the tRNA in the P site is transferred by peptide bond formation to the aminoacyl tRNA in the A site.
6. **Translocation.** THe ribosome moves one codon down the mRNA. The tRNA attached to the polypeptide chain moves into the P site. Uncharged tRNA from P site moves to the E site, where tRNA is ejected. The A site becomes empty again.

### Terminating Translation
- Genetic code includes three stop codons - `UAA`, `UAG`, and `UGA`.
- tRNAs do not work to terminate translation.
  - The translocating ribosome reaches one of the stop codons and a protein called a **release factor** recognizes the stop codon and fills the A site.
- Stop codons found in the 3' region of mRNA.
- Release factors fit tightly into the A site; they have the size and shape of an aminoacyl tRNA.
  - Once in the A site, release factor triggers the hydrolysis of the bond that links tRNA in the P site and frees the polypeptide.
  
#### Process of Terminating Translation
1. **Release factor binds to stop codon.** When the translocating ribosome reaches a stop codon, a protein release factor fills up the A site and breaks the bond linking tRNA to the polypeptide chain.
2. **Polypeptide and uncharged tRNA are released.**
3. **Ribosome subunits separate.** They are ready to attach tot he start codon of another message.

### Polypeptides Are Modified After Translation
- Proteins are not fully formed or functional at the end of translation.

#### Polypeptide Folding
- A protein's function depends on its shape, which depends on how it folds.
- Folding is guided and accelerated by proteins called molecular chaperones.

#### Chemical Modifications
- Eukaryotic proteins are extensively modified even after they are synthesized.
  - e.g. addition of sugars, lipid groups, phosphate groups.

# Video Notes - Transcription and Translation
Andre Ye, 11/14/2020

---

## Video Notes - Transcription and Translation

### DNA Replication and RNA Transcription and Translation
*By Khan Academy. Access [here](https://www.khanacademy.org/science/in-in-class-12-biology-india/xc09ed98f7a9e671b:in-in-the-molecular-basis-of-inheritance/xc09ed98f7a9e671b:in-in-transcription-and-rna-processing/v/rna-transcription-and-translation){:target="_blank"}.*
- DNA's structure makes it suitable for being the molecular basis of heredity.
- Needs to be *replicable*.
  - As a cell divides, the two daughter cells should have the same information.
- Replication: construct another side of DNA based on a given side.
- Expression
  - DNA - talking about a molecule with the double helix structure.
  - A DNA molecule with other proteins to give it a larger structure is a chromosome.
  - A gene is a part of DNA used to code for a certain type of protein.
- RNA is used to convert DNA's information (genotype) into proteins.
- RNA is a messenger between a section of DNA and what goes on outside of the nucleus.
- DNA to mRNA is **transcription.**
  - Adenine pairs with uracil instead of thymine.
- mRNA to proteins is **translation.**
  - Every three bases codes for a specific amino acid.
  - Represents a *codon*.

### The Central Dogma and Phenotypes
*Pearson Media, Figure Walkthrough.*
- DNA to RNA to Protein.
  - Information transfer, not chemical conversion.
- Genotype impacts phenotype.
- Consider: mouse with dark coat.
  - DNA information storage transferred via transcription into mRNA (information carrier).
  - mRNA is translated into proteins.
- The protein is a receptor for a hormone that reduces dark pigment in hair.
- A small change in the phenotype causes a large difference in phenotypes.

### Overview of Protein Syntheis
*Pearson Media*
- Genes are instructions for making proteins.
1. RNA molecule is assembled (pre-mRNA) through transcription.
2. The ends of pre-mRNA are altered in RNA processing and introns are removed (leaving behind exons).
3. The mRNA synthesized then leaves the nucleus and attaches to a ribosome. During trnaslation, the ribosome links the mRNA message and links amino acids together to form a polypeptide.

### The Triplet Nature of the Genetic Code
*Pearson Media*
- Triplet reading frame: if a cell used a sequence to make a protein, all the amino acids should be the same.
- Researchs knew that acridine usually changed the sequence by adding or deleting a nucleotide.
- New sequences code for a different type of amino acids.
  - Destroys functions.
- Acridine can also add a nucleotide base pair to the gene seuqence.
  - The addition causes re-registration of the reading frame.
  - Creates a nonfunctional protein.
- Certain pairs of mutants can restore gene functions.
  - Deletion and addition mutants are crossed, can form a double mutant.
  - The relevant portions are joined.
  - Re-registering the reading frame shows that the two mutations conteract each other.
  - Comparing normal sequence to new sequence, after a short sequence of gibberish, the correct sequence is returned.
- Crick recognized that if the genetic code was in 3 bases, combining three additions should restore the conditions.
- Combining three deletions also restores the conditions.


### Transcription and mRNA Processing
*By Khan Academy. Access [here](https://www.khanacademy.org/science/in-in-class-12-biology-india/xc09ed98f7a9e671b:in-in-the-molecular-basis-of-inheritance/xc09ed98f7a9e671b:in-in-transcription-and-rna-processing/v/transcription-and-mrna-processing){:target="_blank"}.*
- Transcription: going from DNA to mRNA.
- Focusing on genes (genotypes).
- In eukaryotic cells, transcription happens inside the nucleus.
  - DNA to pre-mRNA, processed into mRNA, which leaves the nucleus to be translated.

#### RNA Polymerase and Building RNA
- Protein coding gene: primary actor is RNA polymerase.
- RNA polymerase needs to know where to start. Attaches to a **promoter** sequence of DNA. Indicates where to start.
- Once it attaches, it separates the strands, then is able to code for RNA.
- Only encodes one side (5' to 3').
- This side that it is being interacted with is the *template strand.*
- Will contain the same information as the *coding strand* (opposite the template strand).
- Stops until it reaches a *terminator.*
  - Multiple ways to signal: creates something structurally that forces it to let go.
  - mRNA can form a hairpin (needs the right base pairs), which makes it impair the polymerase and lets go.
  - Might be sequences that part of the polymerase complex recognizes.
- Prokaryote: done. Messenger RNA is ready to go.

#### Eukaryotic Processing
- Things are added: 5' cap (modified guanine) on the 5' end that helps the process.
- Poly-A tail is several adenines on the 3' end.
  - Helps make sure the information is more robust.
  - Makes it less likely the mRNA will be processed.
- In the mRNA sequence, will have *nonsense* sequences, introns.
  - Not coding for a protein.
  - These are processed out (spliced out).
- Leaves behind the *exons* (parts that code for genes).
- Results a *mature* mRNA that codes for the proteins.

### Translation and Protein Synthesis
*By Khan Academy. Access [here](https://www.khanacademy.org/science/in-in-class-12-biology-india/xc09ed98f7a9e671b:in-in-the-molecular-basis-of-inheritance/xc09ed98f7a9e671b:in-in-translation/v/translation-mrna-to-protein){:target="_blank"}.*
- On the strand of DNA, there are sequences called genes.
  - Each code for specific polypeptides or a protein.
- How do you go from information (gene) to a protein?
- Translation: mRNA to proteins. 

- The ribosome is made up of proteins and ribosomal RNA.
  - mRNA does not need to only code information, it can also provide a functional structural role.
- 'top bun' and 'bottom' end, reads it and turns it into a sequence of amino acids.
- Every 3 nucleotides is a codon.
- Information is encoded in the nitrogenous bases.
- AUG: start codon, this is where the protein begins being built by the ribosome.
- Three stop codons (UAA, UAG, UGA).

- tRNA is transfer RNA. Many different tRNAs that combine to each of the amino acids.
- On parts of the tRNA, have *anti-codons* that combine with the actual codons.
  - e.g. AUG and UAC (bound to methionine amino acid).
- Anti-codon matches up with the actual codon on the mRNA.
- The other end of the molecule is where the maino acid is actually bound.

- A-site
  - A-site is where appropriate tRNA initially bounds to the amino acid.
  - Another t-RNA will bond at the A-site, bringing the appropriate amino acid. (A stands for aminoacyl).
  - A peptide bond can form and the ribosome moves.
- P-site
  - The original A-site becomes the P-site.
  - Polypeptide bonds are stored in this area.
- E-site
  - The original P-site becomes the E-site.
  - The tRNA exit at the E-site (disassociate from the amino acids).
- Antibiotics hurt the function of ribosomes of bacterium but not eukaryotes.

---

## Video Notes - Central Dogma, DNA Experiments and Synthesis

### Khan Academy: Central Dogma
[Access here.](#https://www.khanacademy.org/test-prep/mcat/biomolecules/amino-acids-and-proteins1/v/central-dogma-of-molecular-biology-2)
- Explains how we take information and generate a human.
- Credited with discovering this dogma; residue-by-residue transfer of information.

#### DNA Makes RNA Makes Protein
- DNA copies itself through replication.
- DNA can be copied into RNA through transcription.
- RNA can be used to synthesize proteins through translation.

- Each monomer is connected to, at most, one other monomer.
- Specific sequence of each monomer encodes information, and the transfer of information is preserved from DNA to RNA to protein.
- Each polymer is used for the synthesis of the next polymer.
- One could take DNA and see what the RNA would look like, which could be used to determine what the Protein is.

#### Memorizing Terms
- Replication: DNA making copies of itself.
- Transcription: script. Think of it as going from one written form to another form. Both DNA to RNA use nucleotides. Just using one type of alphabet to another.
- Translation: going across languages. Going from nucleotide 'language' to amino acid 'language'.

### Pearson Media: The Hershey-Chase Experiment
- In 1953, Alfred Hershey and Martha Chase confirmed DNA's role in genetics.
- T2 Phage consists of head, sheath, tail, and base plate. DNA is packaged in the head of the virus.
- The Phage attaches to *E. coli* by its tail and injects genetical material. This directs viral enzymes to produce viral offspring.
- The bacterium breaks open and the phages are released.
- T2 consists of protein and DNA. One must contain the genetic material.
- Phages had radioactive DNA or radioactive proteins. 
  - Medium containing radioactive isotope 32P for radioactive DNA.
  - Medium containing radioactive isotope 35S for radioactive proteins.
- Infected *E. coli* with the two types of the phage. Separated phages from rest of the cell. Labelled DNA appeared in host cell but not 'ghosts'. Labelled proteins appeared in 'ghosts' but not host cell.
- Offspring of T2 with labelled DNA were radioactive. Offspring of T2 with labelled protein were not radioactive.
- Convinced the scientific community that DNA was the hereditary material.

### Khan Academy: Semiconservative Replication
[Access here.](https://www.khanacademy.org/science/in-in-class-12-biology-india/xc09ed98f7a9e671b:in-in-the-molecular-basis-of-inheritance/xc09ed98f7a9e671b:in-in-dna-replication/v/semi-conservative-replication)
- Take a piece of DNA and unwind it into its two strands.
- If we were to replicate DNA, what would the end result look like?
- Three choices:
  - Conservative replication. Synthesize a completely new pair.
  - Dispersive replication. End up with 2 pairs of DNA; each pair have some old DNA and new DNA dispersed within the DNA.
  - Semiconservative replication. Each pair has one old strand and one new strand.

#### Meselsohn-Stahl Experiment
- Proved that DNA replication was semi-conservative.

### Bozeman Science: The Meselson-Stahl Experiment
[Access here.](https://www.youtube.com/watch?v=JcUQ_TZCG0w)
- Idea: semi-conservative, conservative, and dispersal models.
- Grew *E. coli*, a bacteria, in *heavy nitrogen* (`15N`). Has an extra neutron.
  - When spun in centrifuge, DNA that contains it will be moved to the bottom. Density of DNA can be used to figure out what is being added.
- Switched heavy nitrogen to normal nitrogen and observed what happened in each generation.
- `14N` (normal nitrogen) and `15N` (heavy nitrogen)
- To make a copy, DNA splits itself in the middle.

### Pearson Media: DNA Replication
- Every cell in you rbody is produced with cell division.
- DNA replication comes from studies of *E. coli*; found in the large intestine.

#### The Replication Fork
- At the origin of replication, two strands of DNA separate to serve as templates.
- Forms a replication bubble. Grows in two directions.
- Forms two replication forks.
- Many proteins work together at the replication fork.
- DNA is unzipped; DNA polymerase builds new strands of DNA.
- Strands in a DNA helix must be made in different ways.

#### Synthesis of the Leading Strand
- Build continuously in the direction of the unzip.
- DNA polymerase builds a new strand of DNA by adding DNA nucleotides one at a time.
- Each nucleotide must pair with its complementary nucleotide.
- Works the same on both leading and lagging strands.

#### Lagging Strand
- Built in pieces.
- Each piece of the lagging strand begins with a short segment of RNA.
- A clamp surrounds the RNA and attaches to DNA polymerase; builds the rest of the piece as DNA.
- When piece is finished, it is released from DNA polymerase.
- A different DNA polymerase removes RNA and replaces it with DNA.
- Cannot finish connecting the pieces. DNA ligase joins the pieces together.

#### Finishing Replication
- Growth of leading and lagging strands grows on both sides until there are two identical DNA molecules.
- Process of replication in bacterial is similar to what happens in our own cells. 

### Pearson Media: Figure Walkthrough, Leading and Lagging Strand Synthesis
- DNA strand synthesis is simple in theory but complex in practice.
- DNA replication requires leading and lagging strands.
- Each strand of DNA is polar. Right or left ends; 5' end and 3' ends (phosphate and hydroxyl groups).
- During DNA synthesis, new building blocks can only be added to the 3' end of the chain.
- In double-stranded DNA, the two must be antiparalel.
- For leading strand, synthesized from 5' to 3' in opposite direction of the template strand.
  - Same direction as the DNA is unwinding.
- Lagging strand is synthesized in 5' to 3' opposite to that of the template; moves in the oppoiste direction of moving replication fork.
- When a lagging strand becomes sufficiently long, it will be used as a template for newly synthesized DNA. 













